src/Jackett.Common/Definitions/torrentlt.yml

```
@@ -119,15 +119,24 @@ settings:
       asc: asc
 
 login:
-  method: cookie
+#   method: cookie
+#   inputs:
+#     cookie: "{{ .Config.cookie }}"
+#   test:
+#     path: lt/torrents.php
+  path: lt/account-login.php
+  method: post
   inputs:
-    cookie: "{{ .Config.cookie }}"
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+  error:
+    - selector: div.klaida
```

Strange. The cookie method is still working for me.
But since they have dropped the ReCaptcha at login we might as well go back to the regular login.
However I don't see the klaida error trap at login,  so I suggest adding error_text as well.
```suggestion
    - selector: div.klaida
    - selector: p.error_text
```